app/helpers/user_helper.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 module UserHelper
-  def tag_name_options      
-    Gutentag::Tag.pluck(:name)
+  def tag_name_options(user)
+    user.predictions.includes(:tags).pluck(:name)
```

Comment:
@jaysonvirissimo I'm pretty sure this is returning the prediction name, not the tag name

Reply:
I was wondering if ActiveRecord would do the right thing here, and it seems like it did, since the specs were making assertions against the literal tag name (for example[ this one](https://github.com/bellroy/predictionbook/pull/203/files#diff-f82478deb69c4a57872a41b8d09ab3122d575bbbc80c9393221e7caa1a6cf3e2R31)).

Reply:
🤔 Yeah you're right, it must have done because there's no `name` attribute on `Prediction`. Weird.